website/docs/components/dropdown/partials/code/component-api.md

```diff
@@ -56,6 +56,14 @@ The Dropdown component is composed of different child components each with their
   <C.Property @name="<[D].Footer>" @type="yielded component">
     `Dropdown::Footer` yielded as contextual component (see below).
   </C.Property>
+  <C.Property @name="<[D].close>" @type="function">
```

Comment:
[suggestion] we could consider adding something also to the "How to use section", in which we explain the fact (maybe not obvious) that the dropdown content is injected in the DOM when rendered, and if a user needs to prevent this behaviour tehy can use the `isOpen` variable (eg. because it triggers an async call to a backend or some similar action that would be better to defer to when the user actually toggles the dropdown)

Reply:
Good point! Added one.